docs/getting-started/first-deployment.md

```diff
@@ -34,10 +33,24 @@ $ fleet database connect prod < database.sql
 
 ## Create your first release
 
+A "release" is a combination of your codes and our base images. Once a release is created, it becomes an inmmutable artefacts and it will be consistent whenenver you load it into an environment.
+
+Depending on which application you are running, the recomended release creation method may be different.
+
+### Magento 1
+
+You can create a relese straight from your git repo. You can just simply run the following command:
 ```bash
 $ fleet release create master
```

Comment:
should note that the "master" is the name of the branch to create it from, they may be using another name for their branch.